docs/development/docs/code-style.md

```diff
@@ -0,0 +1,128 @@
+<!--
```

Comment:
We should remove the `CONTRIBUTING.md` then to prevent documentation duplication.

Reply:
Yes, that's a good point. So do we want to put a short `Contributing` part in our README that basically says that you can find the Contributing documents/styleguide in our development docs?

Reply:
Yes, you can also add a link to the [Github pages](https://dsd-dbs.github.io/capella-collab-manager/).

Reply:
Done. Removed the whole `Code style` part and just added a link to the documentation. The link is  currently broken, since the `code-style` page does not yet exists but I think when we merge this it should be there.